RELEASING.md

```diff
@@ -0,0 +1,231 @@
+
+i. From the root directory,
+    run `cd examples/python`
+
+j. Then,
+    run `pip install -r requirements.txt`
+
+k. Then,
+    run `pip list` and validate that the right version has been installed.
+
+l. run `python3 ../../utils/cluster_manager.py -r 1 -n 3 --cluster-mode`
+    and
+    `python3 ../../utils/cluster_manager.py -r 1 -n 1`
+
+### Trigger the CD workflow for release
+
+Example below is done in python.
+
+a. Edit the file [standalone_example.py](https://github.com/valkey-io/valkey-glide/blob/main/examples/python/standalone_example.py) or [cluster_example.py](https://github.com/valkey-io/valkey-glide/blob/main/examples/python/standalone_example.py) to have the right port in the different functions and run python3.
+
+b. Once the CD workflow and all test have passed, you can release a real version such as `git tag v1.3.0`.
+
+c. Push the tag to the valkey/glide repo so it will trigger the CD action:
+
+d. `git push origin-valkey v1.x.x`
+
+
+2. Update the release notes in Github (**maintainer rights are required**)
+
+a. Go to: https://github.com/valkey-io/valkey-glide/releases
+
+b. Click on "Draft a new release"
+
+## How to use GLIDE's CD workflows
+
+For deployment guide, see the above section [GLIDE - How to release a new version](#glide---how-to-release-a-new-version).
+
+### **How to trigger the CD workflow for Development**
+
+The CD workflow can be triggered in several ways.
+
+1. **By pushing a Version Tag**: Trigger the CD workflow by pushing a version tag in the format "v*.*." or "v.*.*-rc".
+For example, v1.0.0-rc1. When triggered by a push event with a version tag, all package manager CD workflows (e.g., pypi-cd.yml, npm-cd.yml, java-cd.yml) will be initiated.
+The version will be extracted from the tag.
+    ```
+    [13:39:23] ubuntu $ git remote -vv
+    origin  https://github.com/your-repo/valkey-glide (fetch)
+    origin  https://github.com/your-repo/valkey-glide (push)
+    origin-valkey      https://github.com/valkey-io/valkey-glide (fetch)
+    origin-valkey      https://github.com/valkey-io/valkey-glide (push)
+    ```
+
+
+2. **By Workflow Dispatch:** Manually trigger the CD workflow using the "workflow dispatch" option.
+This is done on a specific CD workflow by passing the required version as an input parameter in the format "*.*.*" or "*.*.*-rc*".
+This method will not trigger deployments for other package managers and will not create a tag in the repository.
+
+    ![pypi CD](docs/images/releasing/pypi-CD.png)
+
+3. **Pull Requests:** Pull requests will trigger the CD workflows as "dry runs" without actually publishing to the package managers.
+This ensures that changes do not break the CD logic. Note that PRs affecting the CD workflows (e.g., introducing changes in the CD paths) can only be opened from branches on the
+valkey-io/valkey-glide main repo (rather than from forks) since running the CD, even without publishing, requires access to the repository's secrets.
```

Comment:
```suggestion
`valkey-io/valkey-glide` main repo (rather than from forks) since running the CD, even without publishing, requires access to the repository's secrets.
```